docs/release_notes/v0.6.0/v0.6.0.md

```diff
@@ -1,24 +1,269 @@
+We are happy to announce the official release of KubeBlocks 0.6.0! 🚀 🎉 🎈
+
+This version supports stream computing engines Kafka and Pulsar and supports MySQL read-write splitting, introduces an interactive parameter configuration method.
+
+We would like to extend our appreciation to all contributors who helped make this release happen.
 
 ## Highlights
 
+- KubeBlocks supports Kafka v3.3 
+  - Apache Kafka is an open-source distributed event storage and streaming computing system that provides high reliability, throughput, and low latency for data pipelines, streaming analysis, and data integration. It is widely used in log collection and metric monitoring scenarios. KubeBlocks supports Kafka v3.3, which announced that KRaft meets production requirements, providing better partition scalability and resilience, and saving the additional costs of ZooKeeper. In addition, KubeBlocks also supports pushing data changes from MySQL and PostgreSQL to Kafka, making it easier for users to further process and handle the data.
+- KubeBlocks supports Pulsar v2.11 
+  - Apache Pulsar is an open-source distributed messaging and stream processing platform. It aims to provide scalability, high performance, and reliability to meet the needs of modern data processing and real-time messaging. KubeBlocks supports Apache Pulsar v2.11, and compared to traditional deployment methods, KubeBlocks can automate fault tolerance, scaling, and other day2 operations.
+- KubeBlocks supports MySQL read-write splitting 
+  - Read-write splitting is designed to improve the read-only processing capability of a MySQL database cluster. All write queries are sent to the master node, while read-only queries that do not modify data are distributed to multiple slave nodes. Read-write splitting is used together with the MySQL Raft Group cluster, which automatically detects changes in the master node and uses the current master node of the cluster to achieve failover. Set `read_write_splitting_policy`  as read-write splitting at the global or session level, and the default policy is `LEAST_CURRENT_OPERATIONS`, which routes read-only queries to the slave node with the least active operations. The MySQL Raft Group cluster supports up to 5 nodes.
+
+## Breaking changes
+
+- For KubeBlocks v0.6, newly created Redis clusters need password to connect, which is not supported by v0.5.
+- For KubeBlocks v0.6, the backup and restore function is updated greatly. You need to manually upgrade the 0.5 version cluster to 0.6 and update backupPolicy to use the new function.
+- For KubeBlocks v0.6, password-free login function is canceled for Postgres cluster, and password is required to access the newly created clusters. When restoring, if the cluster is in creating status for a long time, you can check logs to see whether there is `password authentication failed error`, you can update password to fix this.
```

Comment:
For KubeBlocks v0.6, since password-free login function is not supported for Postgres cluster, when restoring cluster upgraded from v0.5, if the new restored cluster is in creating status for a long time, you can check the pod logs to see whether there is password authentication failed error, you can update password to fix this.